migration/idempotent/010-telemetry.sql

```diff
@@ -116,6 +116,12 @@ $$
         SELECT public.approximate_row_count('_ps_trace.span')::TEXT INTO result;
         PERFORM _ps_catalog.apply_telemetry('traces_spans_total_approx', result);
 
+        -- Trace spans processed by jaeger collector will always have an internal attribute named 'internal.span.format' with possible values 'jaeger|zipkin|proto' according to https://github.com/jaegertracing/jaeger/issues/1490. We can use this to infer whether promscale's gRPC remote storage implementation has been used or not.
+        SELECT ARRAY_AGG(value)::TEXT INTO result FROM _ps_trace.tag WHERE key='internal.span.format';
+        IF result IS NOT NULL THEN
+            PERFORM _ps_catalog.apply_telemetry('traces_jaeger_span_types', result);
+        END IF;
```

Comment:
We don't need the NULL check since [apply_telemetry()](https://github.com/timescale/promscale_extension/blob/e94e75347074a69ae514d8e4a1332d0e634ee365/migration/idempotent/010-telemetry.sql#L146) already handles it.

```suggestion
PERFORM _ps_catalog.apply_telemetry('traces_jaeger_span_types', result);
```

Reply:
Actually NULL causes the value to be substituted with 0, I don't want that behaviour. 

https://github.com/timescale/promscale_extension/blob/master/migration/idempotent/010-telemetry.sql#L152-L154